distributed-job-manager/src/test/kotlin/ru/fix/distributed/job/manager/JobDescriptorTest.kt

```diff
@@ -0,0 +1,67 @@
+package ru.fix.distributed.job.manager
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import ru.fix.distributed.job.manager.annotation.DistributedJobId
+import ru.fix.distributed.job.manager.model.JobDescriptor
+import ru.fix.dynamic.property.api.DynamicProperty
+import ru.fix.stdlib.concurrency.threads.Schedule
+
+internal class JobDescriptorTest {
+
+    @Test
+    fun `WHEN jobId notSpecified THEN jobId is full class name with dots replaced by '-' and '$' replaced by '_'`() {
+        val descriptor = JobDescriptor(JobDoesNotDescribingJobId())
+        assertEquals(
+                JobDoesNotDescribingJobId::class.java.name
+                        .replace('.', '-')
```

Comment:
It would be better to compare with actual values that you expect. And do not compute them in the same way as actual implementation does. 
What if you have a mistake in `replace.... ` chain or in `getAnnotation()..value`, if same block of code has a bug, e.g. generate null as a result, test will pass, since production code and test code uses same flawed logic. Assert will endup with `null equals null`. Also explicit values provide better readability. 
`assertEqals("distributed.manger....JobDoesNotDescribingJobid", descriptor.jobId.id`

Reply:
fixed